tc-client/src/main/java/com/tc/object/DistributedObjectClient.java

```diff
@@ -473,13 +428,13 @@ public ClientEntityManager getEntityManager() {
 
   public String getClientState() {
     PrettyPrinter printer = new MapListPrettyPrint();
-    this.communicationStageManager.prettyPrint(printer);
     this.clientEntityManager.prettyPrint(printer);
     return printer.toString();
   }
 
   public void dump() {
     DSO_LOGGER.info(getClientState());
+    DSO_LOGGER.info(ThreadDumpUtil.getThreadDump());
```

Comment:
Was this just for debugging... or is it compensating for losing the `communicationsStageManager`?

Reply:
It was doing a Thread.dumpStack previously which my IDE doesn't like.